documentation/IDTA-01001/modules/ROOT/pages/Spec/IDTA-01001_Metamodel_DataTypes.adoc

```diff
@@ -202,9 +202,9 @@ with max 2024 and min 1 characters
 
 
 ====
-"[anyURI supports a] wide range of internationalized resource identifiers can be specified when an anyURI is called for, and still be understood as URIs per https://www.w3.org/TR/xmlschema11-2/#RFC3986[RFC 3986] and its successor(s)." 
+"[anyURI supports a] wide range of internationalized resource identifiers can be specified when an anyURI is called for, and still be understood as URIs per https://www.w3.org/TR/xmlschema-2/#RFC3986[RFC 3986] and its successor(s)." 
```

Comment:
https://www.w3.org/TR/xmlschema-2/#RFC3986[RFC 3986]
should be changed to:
https://www.w3.org/TR/xmlschema-2/#RFC2396[RFC2396]
(see revised decision in https://github.com/admin-shell-io/aas-specs/issues/299)

Reply:
Done